content/javascript/concepts/statements/statements.md

```diff
@@ -31,6 +31,26 @@ In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.
 
 > **Note:** JavaScript identifiers are case-sensitive. Notice that `x` is not the same as `X`.
 
+
+## codebyte Example
+
+This example shows the output values of the declared variables
+
+```codebyte/javascript
```

Comment:
```suggestion
```codebyte/js
```

Reply:
Hi @dakshdeepHERE. Thanks a lot for your review and suggestions. I earnestly appreciate them. I am currently making the necessary changes. Please kindly confirm that this entry means that I should change all ````codebyte/javascript to codebyte/js.